ktor-client/ktor-client-core/posix/test/ExceptionsTest.kt

```diff
@@ -52,4 +52,4 @@ private fun createResponseException(): ResponseException = ResponseException(obj
         get() = TODO("Not yet implemented")
 
     override fun toString(): String = "FakeCall"
-})
+}, cachedResponseText = "Fake text")
```

Comment:
Could you also add a test when with getting exception, receiving text from response inside that exception and comparing it with the text from exception?